src/components/search/Search.jsx

```diff
@@ -34,15 +34,29 @@ const Search = () => {
         indexName={config.ALGOLIA_INDEX_NAME}
         searchClient={algolia.client}
       >
-        <Configure
-          hitsPerPage={NUM_RESULTS_PER_PAGE}
-          filters={filters}
-          clickAnalytics
-        />
         <div className="search-header-wrapper">
           <SearchHeader containerSize="lg" />
         </div>
-        <SearchResults />
+
+        <Index indexName={config.ALGOLIA_INDEX_NAME}>
+          <Configure
+            hitsPerPage={4}
+            filters={filters}
+            clickAnalytics
+          />
+          { /* A prop will be passed in SearchResults to specify if the search results are courses or programs */ }
+          { /* so it can decide which hit component to use SearchCourseCard or ProgramCard */ }
+          <SearchResults isCourses/>
```

Comment:
Might be because it's a WIP PR, but edx.org/search has 4 "Programs" and 20 "Courses" shown by default, with "Programs" coming before "Courses"

Reply:
Yes, just wanted to get your feedback on the approach first before making any design changes. But we are all set now, I have added a new image in the description of what the search page will look like.

Reply:
https://drive.google.com/file/d/1lH9QHEPuU0pl_ZwFuxxp7gzASVLcFAsk/view?usp=sharing